modules/serverless-jaeger-config.adoc

```diff
@@ -16,27 +11,20 @@ To configure Jaeger for use with {ServerlessProductName}, you will need:
 
 .Procedure
 
-. Create and apply a Jaeger custom resource YAML file that contains the following sample YAML:
+. Create and apply a `Jaeger` resource YAML file that contains the following:
```

Comment:
Why is `kind: Jaeger` not a custom resource? Is this something specific to Serverless?

Reply:
It is a custom resource but idk the additional word just seemed unnecessary to me, I can add it back in though